core/server/services/auth/session/index.js

```diff
@@ -49,3 +49,4 @@ module.exports.createSessionFromToken = sessionFromToken({
 });
 
 module.exports.sessionService = sessionService;
+module.exports.sessionStore = expressSession.sessionStore;
```

Comment:
I was thinking about this bit and I think a cleaner approach here would be exposing a separate method on the `sessionsService` like `clear()` that would call the `store.clear()`. "Store" is an internal implementation detail of this module, don't think controllers should know about it. The other upside is having way smaller exposed API surface with a method vs whole Store instance. Wdyt?

Reply:
I didn't think about this approach, I'm looking into it now 👀

Reply:
After a couple of attempts I think it's cleaner not to bring the concept of express-session into our already-extracted session library.
So I've simply reduced the surface area of what's exposed by only exposing a promisified version of `store.clear()`, I think it's the best of both worlds

Reply:
Love it!